tests/operator/operator.go

```diff
@@ -2971,6 +2971,10 @@ spec:
 			const expectedSeccompProfilePath = "/proc/1/root/var/lib/kubelet/seccomp/kubevirt/kubevirt.json"
 
 			enableKubevirtProfile := func(enable bool) {
+				By("Removing profile if present")
+				_, err := tests.ExecuteCommandInVirtHandlerPod(nodeName, []string{"/usr/bin/rm", "-f", expectedSeccompProfilePath})
```

Comment:
Is `nodeName` always set if you move it above?

Reply:
oops, you're right I forgot to move that line as well, thanks!